model.py

```diff
@@ -4355,16 +4292,62 @@ def update_quality(self):
                          ((self.voted_quality or 0) * votes_for_quality))
         self.quality = total_quality / float(total_weight)
 
-    def set_representation(self, media_type, content, uri=None,
```

Comment:
I took this method out. Nobody calls it and it's clearly broken. It refers to a `_db` variable that doesn't exist and calls a nonexistent `.generic_uri` method.